### PR TITLE
OD-258 [Feature] Filter logic: Number is between

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 
## Issue
OD-258 https://weboo.atlassian.net/jira/software/projects/OD/boards/9?selectedIssue=OD-258
## Description
Added new logic for `Filter logic: Number is between`, which are filtering the list between two values

base on commit - 06e0535b9bc26a97e59a63daf5a8998e371c6f0a
## Screenshots/screencasts
http://somup.com/cYlU6r2MTV
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko @ivandevupp 
## Notes 
link to the previously approved PR - https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/473

The PR's should be merged in the sequence OD-252,254,258

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -1929,7 +1935,6 @@ var DynamicLists = (function() {
       $filterAccordionContainer.append($newPanel);
 
       if (data.logic === 'empty' || data.logic === 'notempty') {
-        $newPanel.find('.panel-title-text .value, #value-dash, #filter-value-type').addClass('hidden');
```
Comment (tonytlwu) [inline js/interface.js:1932]: Why is this removed? If you look at [the previously approved PR](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/473), the `interface.js` file had this line https://github.com/AndrRyaz/fliplet-widget-dynamic-lists/blob/7eb19e2dc9f7b5c0de46d1b57945f6d9c70f0a1e/js/interface.js#L1940

Comment (AndrRyaz) [reply js/interface.js]: @tonytlwu, when i created a new branch from base master and update them from upstream, this changes was already implemented, but your right this line should be, fixed

**Review by tonytlwu — DISMISSED** [review]
@sofiiakvasnevska This is ready for testing

Risk: High
Affected: LFDs with filters configured in Studio
Verify: LFD filter configuration works as expected

Squash and merge: Add filter logic for "number is between"
